ui-tests/cypress/integration/pages/management.spec.js

```diff
@@ -348,6 +348,23 @@ describe('Page Management', () => {
         currentPage.getContent().getOwnerGroupButton().should('be.disabled');
       });
 
+      it('Change a page\'s owner group - not allowed', () => {
+        page.code = generateRandomId();
+        page.title    = {
+          en: generateRandomId(),
+          it: generateRandomId()
+        };
+        cy.pagesController()
+          .then(controller => controller.addPage(page.code, page.title.en, page.ownerGroup.code, page.template, page.parentCode))
+          .then(() => pageToBeDeleted = true);
+
+        currentPage = openManagementPage();
+        currentPage = currentPage.getContent().getKebabMenu(page.code).open().openEdit();
+
+        currentPage.getContent().getOwnerGroupButton().click({ force: true });
```

Comment:
Why it is needed _force: true_?
If it is 'cause the button is disabled (hence there will be no dropdown), we can simply check that...

Reply:
The associated bug was that the dropdown list still shows when the user clicks it even if it's disabled. Hence, `force: true` is added to make sure that the list doesn't show when the user clicks it.

Reply:
I also noticed this particular field that when using it with `click` won't work unless adding `force: true`. I noticed this because I am working on a related task.

Reply:
ok, got it, if it is part of the test it makes sense; I just wanted to be sure it was not used because the element was not clearly identified